protocol.html

```diff
@@ -380,13 +380,17 @@ <h3 property="schema:name">Overview</h3>
 
                   <p>The Solid Notification protocol defines a mechanism for client applications to discover notification services available for any given resource.</p>
 
-                  <p>The following diagrams show the high-level interactions involved in discovering Notification Channels and Subscription Services. A Subscription Service allow Subscription Clients to request a customised Notification Channels.</p>
+                  <p>The following diagram shows the high-level interactions involved in discovering Notification Channels and Subscription Services.</p> 
 
                   <figure id="solid-notifications-flow-discovery" rel="schema:hasPart" resource="#solid-notifications-flow-discovery-subscription">
                     <img rel="schema:image" src="notifications-flow-discovery.mmd.svg" width="800" />
 
                     <figcaption property="schema:name">Solid Notifications Flow Discovery</figcaption>
                   </figure>
+                  
+                  <p>A Subscription Service allows Subscription Clients to request a customised Notification Channel.</p>
+                  
+                  <p>The following diagram shows the high-level interactions involved in establishing a customised Notification Channel.</p>
```

Comment:
Q is whether Solid docs are meant to use US or UK spelling variations. (W3C uses US.)
```suggestion
                  <p>A Subscription Service allows Subscription Clients to request a customized Notification Channel.</p>
                  
                  <p>The following diagram shows the high-level interactions involved in establishing a customized Notification Channel.</p>
```

Reply:
The original PR remains consistent with the document. This suggestion breaks it.

We've discussed the spelling a few times in the past. The manual of style('s Spelling) is not a requirement AFAIK and have not seen it strictly applied. It is not required for Solid specs either. W3C spellchecker supports English in general, and not limited to a specific sub-dictionary.

Can we move on?

Reply:
Given that we're operating as a [W3C CG](https://www.w3.org/groups/cg/solid/), I think we should be conforming to the [W3C Manual of Style](https://www.w3.org/Guide/manual-of-style/), and to [11.2 Spelling](https://www.w3.org/Guide/manual-of-style/#Spelling) in particular.

The [W3C Manual of Style](https://www.w3.org/Guide/manual-of-style/) would probably benefit from some attention from the [I18n](https://www.w3.org/groups/wg/i18n-core) folks, and I'd be happy to adjust my editing to such an update.

It is more than regrettable that the [W3C Manual of Style](https://www.w3.org/Guide/manual-of-style/) says (more than once!) to use US English spelling and then points to a W3C spellchecker that only supports a generic English which appears to accept both US and UK spellings, but at least that checker only works *on w3.org URIs*.  (It doesn't do anything with GitHub pages, nor within my local browser or other editor, where I am set up to use US English, as advised.)  The W3C spellchecker is thoroughly problematic, as it can easily lead to one document that contains a mix of spellings _of the same word_, which is only confusing.

If the Solid Project is to have a different Style than W3C, I'd like that to be codified somewhere, so there's no question of whether spellings should be `ou` or `o`, or `s` or `z`, etc.

You are of course free to reject this requested/suggested change. I would suggest, in that case, that an issue should be logged on the specification repo, toward a definitive conclusion going forward.

Reply:
tracked in https://github.com/solid/process/issues/318 which can be transferred to another repo if needed